crates/uv-cli/src/lib.rs

```diff
@@ -4810,10 +4810,9 @@ pub enum PythonCommand {
     /// Python versions are installed into the uv Python directory, which can be retrieved with `uv
     /// python dir`.
     ///
-    /// A `python` executable is not made globally available, managed Python versions are only used
-    /// in uv commands or in active virtual environments. There is experimental support for adding
-    /// Python executables to a directory on the path — use the `--preview` flag to enable this
-    /// behavior and `uv python dir --bin` to retrieve the target directory.
+    /// By default, Python executables are added to a directory on the path as `pythonX.y`. To
```

Comment:
capitalization (or, alternatively, just write `python3.x`)

Reply:
This is intentional to indicate "MAJOR"and "minor", but the point is moot because I've edited this to just use a concrete `python3.13` example and haven't pushed it yet to avoid CI churn.